src/main/kotlin/com/simplymerlin/minigameserver/minigame/blockparty/phase/RoundPhase.kt

```diff
@@ -130,11 +131,15 @@ class RoundPhase(val game: BlockPartyGame) : StateSeries() {
         override var time = 5
 
         override fun onStart() {
+            val batch = AbsoluteBlockBatch()
             round.playingField.forEach {
                 if (round.instance.getBlock(it) != round.chosenBlock.block) {
-                    round.instance.setBlock(it, Block.AIR)
+                    batch.setBlock(it, Block.AIR)
                 }
             }
+            batch.apply(round.instance) {
+                println("Batch complete!")
```

Comment:
can we get rid of this println? and everywhere else where it shows up

Reply:
And just replace it with nothing?

Reply:
Or perhaps we should move logic that occurs afterwards to be moved into the callback